pkg/terraform/terraform.go

```diff
@@ -127,7 +121,10 @@ func NewTerraformCLI(config *TerraformCLIConfig) (*TerraformCLI, error) {
 // `terraform workspace new <name>`
 func (t *TerraformCLI) Init(ctx context.Context) error {
 	var wsCreated bool
+	var out bytes.Buffer
 
+	t.tf.SetStdout(&out)
```

Comment:
I'm not sure about forcing this on the caller. Should it be an argument? 
`func (t *TerraformCLI) Init(ctx context.Context, w writer) error {` etc

Reply:
and then you can `t.tf.SetStderr(&w)` or something similar.

Reply:
Wasn't there a cool method that let you specify your stdout at one point?

Reply:
@jasonBirchall , I have updated to pass the Write as an argument but still prints the output using the terraform method. let me know if that looks ok